### PR TITLE
General overhaul

### PR DESCRIPTION
**Pull request by kfichter**
This is a big one!

### CODE REVIEW

**Comment by kfichter** [comment]
Latest deploy preview: https://deploy-preview-18--thirsty-swanson-df3e28.netlify.app/

File: src/docs/developers/integration.md [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+
+You might also want to take the following two steps depending on your specific needs:
+
+1. **(Optional) Deploy and test against L2 Geth:** get those contracts running on a local instance of the full L2 system with our modified version of [go-ethereum](https://github.com/ethereum-optimism/go-ethereum).
+2. **(Optional) Create bridges between L1 and L2:** get L1 and L2 communicating where necessary, such as adding deposits/withdrawals.
+
+## Compiling and Testing Contracts
+
+### Setup and Resources
+// TODO: Add a better "required tooling" section.
```
Comment (platocrat) [inline src/docs/developers/integration.md:38]: Remove the comment since I turned it into an issue

File: src/docs/developers/integration.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+You probably want to deploy to testnet before heading over to mainnet (good idea, tbh).
+Our primary L2 testnet is currently deployed on top of Ethereum's [Kovan](https://kovan.etherscan.io) network.
+We sometimes run other testnets too. 
+Here's a current list of our testnet endpoints (will be updated when things change):
+
+| L1 Network                          | L2 Network | HTTP Endpoint                                          | Websocket Endpoint                                               |
+| ----------------------------------- | ---------- | ------------------------------------------------------ | ---------------------------------------------------------------- |
+| [Kovan](https://kovan.etherscan.io) | OE Kovan 1 | [https://kovan.optimism.io](https://kovan.optimism.io) | [https://kovan.optimism.io:8546](https://kovan.optimism.io:8546) |
+
+// TODO?
```
Comment (platocrat) [inline src/docs/developers/integration.md:157]: turned this into another issue

File: src/docs/resources/tutorials.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+:::
+
+---
+## [*The "Official" Optimistic Ethereum Tutorial*](../tutorial/)
+
+* By: [Optimism PBC](http://optimism.io)
+
+The "official" introductory tutorial by the Optimism team.
+Note, that this tutorial is _short and sweet_.
+For more thorough guides, please refer to other the resources below or to the Protocol/Integration Docs (check the sidebar).
```
Comment (platocrat) [inline src/docs/resources/tutorials.md:24]: should be "Protocol/Developer Docs"

**Review by platocrat — CHANGES_REQUESTED** [review]
Besides those comments, LGTM!

**Review by platocrat — APPROVED** [review]

**Comment by TransmissionsDev** [comment]
lol i love how we're merging kevins stuff

**Pull request merged**